tools/environment-setup.sh

```diff
@@ -14,9 +14,15 @@ if ! grep -q '^name: snapcraft$' snap/snapcraft.yaml; then
 fi
 
 # Create the container
-lxc init ubuntu:16.04 snapcraft-dev
-lxc config set snapcraft-dev raw.idmap "both $UID 1000"
-lxc start snapcraft-dev
+if ! lxc list | grep -q "snapcraft-dev"; then
```

Comment:
suggested improvement to improve accuracy:

`if ! lxc info snapcraft-dev >/dev/null 2>&1`

Reply:
Thanks! I will update the PR

Reply:
Thank you for the updates!